### DISCUSSION TITLE
Wix v4 - After Wix4 installation, are these Versions of Candle.exe and Light.exe valid versions

### DISCUSSION THREAD

**Discussion by garevaul** [wixtoolset/issues#7354] [Questions]
After Wix4 installation, are these Versions of Candle.exe (v3.11.2) and Light.exe (v3.11.2) valid versions ? Or is there any Later Version than these ?

![Wix4(1)](https://user-images.githubusercontent.com/15125247/229521218-fac8cf49-6615-4db8-aa92-b000ceafcf13.png)

![Wix4(2)](https://user-images.githubusercontent.com/15125247/229521285-f24cbfb0-ac3c-4f8d-baa6-074a8d461b98.png)

**Comment by robmen** [comment]
That's WiX v3.11.2. It does not support the `--version` switch.

**Comment by garevaul** [comment]
Rob,

You mean Wix4 does not use Candle and Light ?

If it does, then MAYBE the Candle and Light currently being accessed by it is the v3.11.2 one.

Please see the PoweShell screenshots I have uploaded.

Regards 

@garevaul

**Reply by wmanning** [reply]
My understanding is in wix 4 there are no separate candle and light executable.  You use the one wix exe with different command line arguments.

**Reply by BMurri** [reply]
@wmanning your understanding is correct. Both "candle" and "light" are referenced using the build command of the `wix` tool (which itself can be accessed via the `dotnet` tool if it's installed into its ecosystem). There're no longer `candle` or `light` tools in v4.

**Reply by garevaul** [reply]
Thanx guys

Regards
@garevaul

**Comment by robmen** [comment]
@garevaul The very [first episode of the Deployment Dojo](https://www.youtube.com/live/-Y7dOtciOJw?feature=share) walks through this. Check it out to get a better understanding.

**Reply by garevaul** [reply]
Thanx a lot @robmen. The session covers a lot of things.
